pkg/model/v0/decoder.go

```diff
@@ -0,0 +1,61 @@
+package v0
+
+import (
+	"fmt"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/grafana/tempo/pkg/model/trace"
+	"github.com/grafana/tempo/pkg/tempopb"
+)
+
+const Encoding = ""
```

Comment:
I split the interface to hide Marshal from the outside world and now the interface is called "Decoder". Agree that calling everything "Encoding" was getting weird. This may help with this const name. LMK